crates/bevy_ecs/src/storage/table.rs

```diff
@@ -237,27 +252,66 @@ impl Column {
     }
 
     #[inline]
-    pub fn get_ticks(&self, row: usize) -> Option<&UnsafeCell<ComponentTicks>> {
-        self.ticks.get(row)
+    pub fn get_added_ticks(&self, row: usize) -> Option<&UnsafeCell<Tick>> {
+        self.added_ticks.get(row)
+    }
+
+    #[inline]
+    pub fn get_changed_ticks(&self, row: usize) -> Option<&UnsafeCell<Tick>> {
+        self.changed_ticks.get(row)
+    }
+
+    #[inline]
+    pub fn get_ticks(&self, row: usize) -> Option<ComponentTicks> {
```

Comment:
Comments on these methods about how the `row`  values works and why this might return `None` would be nice. Won't block on it though; this is no worse than before.

Reply:
I'll go ahead and make a separate PR for this.